tests/system/small/test_pandas.py

```diff
@@ -424,6 +424,31 @@ def test_cut_default_labels(scalars_dfs):
     )
 
 
+def test_cut_numeric_breaks(scalars_dfs):
+    scalars_df, scalars_pandas_df = scalars_dfs
+
+    breaks = [0, 5, 10, 15, 20, 100, 1000]
```

Comment:
can we also add a test case (parameterize) for float values?

Reply:
Good point, added, thanks. Also fixed mixing ints and floats.